doc/source/examples/index.rst

```diff
@@ -34,11 +34,12 @@ Here are a series of examples using MAPDL with ``ansys-mapdl-core``.
 
 .. toctree::
     :hidden:
-    :maxdepth: 3
+    :maxdepth: 4
```

Comment:
The max depth is related to until what level of headings (title, section, subsection ...) is going to show in that table of contents (toctree)

```suggestion
    :maxdepth: 3
```